util/design/README.md

```diff
@@ -163,11 +163,50 @@ TODO
 
 ## LFSR Coefficient Generator Tool
 
-TODO
+The `get-lfsr-coeffs.py` script is used to fetch a list of primitive polynomials for Galois-type LFSRs.
+These coefficients are downloaded from [https://users.ece.cmu.edu/~koopman/lfsr/](https://users.ece.cmu.edu/~koopman/lfsr/).
+When used with the `--fib` switch, the script instead downloads polynomials for Fibonacci-type LFSR.
+The script downloads text files containing the first 100 primitive polynomials for LFSR lengths ranging from 4 to 64 and places them into a temporary folder.
+The script also produces an output file containing the SystemVerilog template with LFSR polynomials for lengths 4 to 64.
+This template contains exactly one polinomial for each LFSR length, this is always the first polynomial listed in the corresponding file.
 
 ## LFSR Seed Generator Tool
 
-TODO
+The `gen-lfsr-seed.py` script creates a SystemVerilog template for LFSR parameters i.e. width, seed and the permutation of the output bits.
```

Comment:
Nit: Missing comma -> `...parameters, i.e., width, seed and the...`

Reply:
Also, it should be `default seed` rather than `seed` because the HW IP block can be reseeded at runtime.

Reply:
Agreed, I've made the changes